config/servicemappings/alloydb.yaml

```diff
@@ -142,6 +143,16 @@ spec:
             version: v1beta1
             kind: AlloyDBCluster
           targetField: name
+        - tfField: secondary_config.primary_cluster_name
+          description: |-
+            "(Required) Name of the primary cluster must be in the format
```

Comment:
Nit: Maybe remove the double quotes? We don't usually have those in the field description.

Reply:
Removed double-quotes